akka-docs/src/main/paradox/distributed-pub-sub.md

```diff
@@ -17,32 +17,7 @@ To use Distributed Publish Subscribe you must add the following dependency in yo
 
 ## Introduction
 
-How do I send a message to an actor without knowing which node it is running on?
-
-How do I send messages to all actors in the cluster that have registered interest
-in a named topic?
-
-This pattern provides a mediator actor, `akka.cluster.pubsub.DistributedPubSubMediator`,
-that manages a registry of actor references and replicates the entries to peer
-actors among all cluster nodes or a group of nodes tagged with a specific role.
-
-The `DistributedPubSubMediator` actor is supposed to be started on all nodes,
-or all nodes with specified role, in the cluster. The mediator can be
-started with the `DistributedPubSub` extension or as an ordinary actor.
-
-The registry is eventually consistent, i.e. changes are not immediately visible at
-other nodes, but typically they will be fully replicated to all other nodes after
-a few seconds. Changes are only performed in the own part of the registry and those
-changes are versioned. Deltas are disseminated in a scalable way to other nodes with
-a gossip protocol.
-
-Cluster members with status @ref:[WeaklyUp](typed/cluster-membership.md#weakly-up),
-will participate in Distributed Publish Subscribe, i.e. subscribers on nodes with
-`WeaklyUp` status will receive published messages if the publisher and subscriber are on
-same side of a network partition.
-
-You can send messages via the mediator on any node to registered actors on
-any other node.
+For the full documentation of this feature and for new projects see @ref:[Distributed Publish Subscribe - Introduction](typed/distributed-pub-sub.md#introduction).
```

Comment:
I think we should keep the Classic docs of this feature as is, because we want to introduce something new for Typed later, and that is not ready yet.

Reply:
Reverted.